src/index.native.scroll.js

```diff
@@ -103,7 +103,9 @@ class SwipeableViews extends React.Component {
       index,
     } = nextProps;
 
-    if (typeof index === 'number' && index !== this.props.index) {
+    // Sometimes the index can be the same, but the slide
+    // may have changed, so just always set it
+    if (typeof index === 'number') {
```

Comment:
I'm not sure about this change.
I feel like it's for updating `offset.x` that really on `this.state.viewWidth` and `index`.
The index will be the same, so what could have changed is `this.state.viewWidth`. Should it be already handled by `onLayout`.